test/dlc_tests/conftest.py

```diff
@@ -192,7 +192,7 @@ def delete_ssh_keypair():
     else:
         # Using private AMI, the EBS volume size is reduced to 28GB as opposed to 50GB from public AMI. This leads to space issues on test instances
         # TODO: Revert the configuration once DLAMI is public
-        params["BlockDeviceMappings"] = [{"DeviceName": "/dev/sda1", "Ebs": {"VolumeSize": 50,}}]
+        params["BlockDeviceMappings"] = [{"DeviceName": "/dev/sda1", "Ebs": {"VolumeSize": 90,}}]
```

Comment:
Necessary for Neuron inference test.

Reply:
Is the `90Gb` size requirement for neuron as this is a temp change. Once reverted, the instance will have `50Gb` by default